.github/workflows/ci.yml

```diff
@@ -4,39 +4,47 @@ on:
     branches: [main]
     tags: ["v*"]
 
+env:
+  CARGO_INCREMENTAL: 0
+  CARGO_NET_RETRY: 10
+  RUST_BACKTRACE: short
```

Comment:
I'm pro-always-enable-backtraces, but were we getting backtraces on CI builds?

Reply:
You can get them during tests